### PR TITLE
Document proposed alt_text properties.

### PR DESCRIPTION
**Pull request by transplier**
## Proposed change

Added `alt_text` property support to cards which feature images. Please see https://github.com/home-assistant/frontend/pull/14405

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/frontend/pull/14405
- Link to parent pull request in the Brands repository:  n/a
- This PR fixes or closes issue: fixes #14178 in frontend

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
There are unrelated commit in this PR, probably caused by this PR being branched from the `current` branch.

Please git rebase your commit onto the `next` branch to clean this PR up.

Thanks 👍 

../Frenck

**Comment by transplier** [comment]
Shoot, yes, silly of me. Fixing.

File: source/dashboards/header-footer.markdown [hunk 1]
```diff
@@ -40,7 +40,10 @@ double_tap_action:
   required: false
   description: Action taken on double tap. See [action documentation](/dashboards/actions/#double-tap-action).
   type: map
-{% endconfiguration %}
```
Comment (steverep) [inline source/dashboards/header-footer.markdown]: Need to add this line back.  Probably why the checks are failing

File: source/_dashboards/picture-entity.markdown [hunk 2]
```diff
@@ -38,6 +38,10 @@ state_image:
   required: false
   description: "Map entity states to images (`state: image URL`, check the example below)."
   type: map
+alt_text:
```
Comment (steverep) [inline source/_dashboards/picture-entity.markdown]: Let's get rid of this and create the alt text per my comment on the frontend PR.

File: source/_dashboards/picture.markdown [hunk 3]
```diff
@@ -21,6 +21,10 @@ image:
   required: true
   description: "The URL of an image. When you want to store images in your Home Assistant installation use the [hosting files documentation](/integrations/http/#hosting-files). After storing your files, use the `/local` path, for example, `/local/filename.jpg`."
   type: string
+alt_text:
+  required: false
+  description: "Image alternate text. This is useful for users of assistive technology."
```
Comment (steverep) [inline source/_dashboards/picture.markdown]:
```suggestion
  description: "Alternative text for the image. This is useful for users of assistive technology."
```

File: source/dashboards/header-footer.markdown [hunk 4]
```diff
@@ -40,7 +40,10 @@ double_tap_action:
...
+alt_text:
+  required: false
+  description: "Image alternate text. This is useful for users of assistive technology."
```
Comment (steverep) [inline source/dashboards/header-footer.markdown]:
```suggestion
  description: "Alternative text for the image. This is useful for users of assistive technology."
```

**Review by steverep — CHANGES_REQUESTED** [review]

File: source/dashboards/header-footer.markdown (hunk 1 above)
Comment (transplier) [reply source/dashboards/header-footer.markdown]: That was.... silly of me :/

File: source/dashboards/header-footer.markdown [hunk 5]
```diff
@@ -40,6 +40,10 @@ double_tap_action:
   required: false
   description: Action taken on double tap. See [action documentation](/dashboards/actions/#double-tap-action).
   type: map
+alt_text:
```
Comment (steverep) [inline source/dashboards/header-footer.markdown]: Can we move this above the actions?  Makes more sense right after the image source.

**Review by steverep — APPROVED** [review]

**Pull request merged**